eva_cttv_pipeline/config.py

```diff
@@ -1,5 +1,34 @@
 local_schema = "resources/schema_local"
 
+
 sparqlep = ""
 sparql_user = ""
 sparql_password = ""
+
+
+#########################################
+# clinvar_to_evidence_strings.py settings
+#########################################
+# cellbase
+BATCH_SIZE = 200
+HOST = 'www.ebi.ac.uk'
+
+# output settings
+EVIDENCESTRINGSFILENAME = 'evidence_strings.json'
```

Comment:
The consistency among constants names, and also their readability, could be improved by using underscores and whole words in all of them.